Makefile

```diff
@@ -65,6 +65,10 @@ test-skip-proptests: ## Runs all tests, except property-based tests
 test-loom: ## Runs all loom-based tests
 	RUSTFLAGS="--cfg loom" cargo nextest run --cargo-profile test-release --features testing -E 'test(#*loom)'
 
+.PHONY: test-package
+test-package: ## Tests specific package: make test-package package=miden-vm
+	$(DEBUG_ASSERTIONS) cargo nextest run --cargo-profile test-release --features testing -p $(package)
```

Comment:
Nice! A question: would we achieve the same result if we add `executable` feature? e.g.,:
```shell
$(DEBUG_ASSERTIONS) cargo nextest run --cargo-profile test-release --features testing,executable
```

Reply:
The thing that I added will run the integration tests for just one package as a convenience to avoid having to run all packages. Adding feature "executable" is required to run the unit tests inside the binary crate of `miden-vm` package. I created issue #1501 with the question regarding whether adding "executable" is the right way fix the problem of unit tests in `miden-vm` not running when using `make tests`